pypeit/par/pypeitpar.py

```diff
@@ -686,7 +686,8 @@ class Coadd1DPar(ParSet):
     def __init__(self, ex_value=None, flux_value=None, nmaskedge=None,
                  sn_smooth_npix=None, wave_method=None, samp_fact=None, ref_percentile=None, maxiter_scale=None,
                  sigrej_scale=None, scale_method=None, sn_min_medscale=None, sn_min_polyscale=None, maxiter_reject=None,
-                 lower=None, upper=None, maxrej=None, sn_clip=None, nbest=None, sensfuncfile=None, coaddfile=None):
+                 lower=None, upper=None, maxrej=None, sn_clip=None, nbest=None, sensfuncfile=None, coaddfile=None,
+                 mag_type=None, filter=None, filter_mag=None, filter_mask=None):
```

Comment:
To me the mags for renormalizing should be parameters not passed in via parsets, but I think this would require another way to think about file parsing. I will tolerate it for now.

Reply:
Yup.